docs/getting_started/requirements.md

```diff
@@ -41,19 +37,23 @@ Supported setups are those we perform automated testing on. For security and per
 -   Web Servers:
     -   Apache 2.4, with required modules `mod_rewrite`, `mod_env` and recommended: `mod_setenvif`, `mod_expires`
         - event MPM is recommended, if you need to use _prefork_ you'll also need the `mod_php` module
-    -   Nginx 1.10, 1.12, 1.14
+    -   Nginx 1.12, 1.14
 -   DBMS
-    -   MySQL 5.5.3 and higher, 5.6\*, 5.7\*
-    -   MariaDB 5.5 (new enough to support `utf8mb4` character set), 10.0, 10.1, 10.2\*, 10.3\*
+    -   MySQL 5.7\* or 8.0\* \** 
+    -   MariaDB 10.0, 10.1, 10.2\*, 10.3\*
+    - PostgreSQL 10+
```

Comment:
@SylvainGuittard no, this is fine. Seems I need to upgrade my development server as well :)

Reply:
@alongosz we could say 9.6+ if that is what you are using, it's one of the versions available with RHEL 8.x still so it can make sense.

But we should have a recommendation, so QA can primarily test what we recommend.

Reply:
@andrerom from my POV it's best to support newest versions (btw. there's PostgreSQL 11 already), but only due to performance. Version 10 introduces parallel index scanning, so it might be the "go to" database for big enterprise databases with millions of Content items.

For Ubuntu it turned out to be quite simple. I just switched already installed and running PostgreSQL 10 instance to bind to default port ;-)

My question is - do we know for sure that some Customers will have problem upgrading to PostgreSQL 10? If so, then I'm fine with `9.6+`. It doesn't require adjustments on our end as the differences exist in areas we don't use because of cross-DBMS support. Though performance matters also.

Reply:
all the recommended distros here has 10, and we haven't supported postgres in long while so we don't officially have any customers on this ;)

so `10+` sounds like a good base here, it was released on `2017-10-05` so should be pretty safe by the time people start to use 2.5LTS later this year.